media/css/products/vpn/vpn-info.scss

```
@@ -0,0 +1,19 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../protocol/css/includes/lib';
+
+.c-section-heading {
+    margin-top: $layout-sm;
+}
+
+.c-list-content {
+    margin-left: $layout-xs;
```

I know these pages are only English right now but this would be a good place to use the [bidi mixin](https://github.com/mozilla/protocol/blob/main/src/assets/sass/protocol/includes/mixins/_bidi.scss) to handle both LTR and RTL.

```suggestion
    @include bidi((
        (margin-left, $layout-xs, 0),
        (margin-right, 0, $layout-xs),
     ));
```

---

oh! nice catch